__main__.py

```diff
@@ -35,23 +44,31 @@ def main():
     env = simpy.Environment()
 
     server_conf = get_config('config/server.ini', 'server sleep_time-0.00001')
-    if scenario == 'control':
-        gc_conf = get_config('config/gc.ini', 'gc sleep_time-0.00001 threshold-0.9')
-        gci_conf = get_config('config/gci.ini', 'gci sleep_time-0.00001 threshold-0.7 check_heap-2 initial_eget-0.9')
-        server = ServerWithGCI(env, 1, server_conf, gc_conf, gci_conf)
 
-    elif scenario == 'baseline':
-        gc_conf = get_config('config/gc.ini', 'gc sleep_time-0.00001 threshold-0.75') # Should change some configuration
-        server = Server(env, 1, server_conf, gc_conf)
+    servers = list()
+    for i in range(NUMB_OF_SERVER):
+        if scenario == 'control':
+            gc_conf = get_config('config/gc.ini', 'gc sleep_time-0.00001 threshold-0.9')
+            gci_conf = get_config('config/gci.ini', 'gci sleep_time-0.00001 threshold-0.7 check_heap-2 initial_eget-0.9')
+            server = ServerWithGCI(env, i, server_conf, gc_conf, gci_conf, log_path)
+
+        elif scenario == 'baseline':
+            gc_conf = get_config('config/gc.ini', 'gc sleep_time-0.00001 threshold-0.75') # Should change some configuration
+            server = Server(env, i, server_conf, gc_conf, log_path)
+
```

Comment:
What if the scenario is neither control nor baseline? It should error raise an error, right?

Reply:
Yes! I agree. I had let to think about what raise later. Thanks for remember.